kitsune/sumo/jinja2/includes/readonly_annoucement.html

```diff
@@ -0,0 +1,11 @@
+{% trans
+    irc_channel="https://client02.chat.mibbit.com/?server=irc.mozilla.org&channel=%23sumo",
+    reddit="http://reddit.com/r/firefox",
+    twitter="https://twitter.com/firefox" %}
+  Some maintenance on the site is happening to cause the site to go
+  into read-only mode for a bit. If you are trying to ask a question
+  and an article does not solve your issue, please find one of our
+  partner sites for more help: <a href="{{ twitter }}">@firefox</a> on
+  Twitter, <a href="{{ reddit }}">/r/firefox</a> on Reddit, we will
+  have community standing by. Or live on our IRC channel <a href="{{ irc_channel }}">#sumo</a>.
```

Comment:
This last sentence isn't a complete sentence. Otherwise 👍

Reply:
Agree, i'll ping sumo team in case they want to update